File: Tree/Tree Traversal/Postorder Traversal (Recursive)/sol.c
#include <stdio.h>
#include <malloc.h>


// declaring node of Binary-Tree
struct BinaryTreeNode {
    struct BinaryTreeNode *left;
    int info;
    struct BinaryTreeNode *right;
};


struct BinaryTreeNode *root=NULL;



// function for postOrder Traversal (Recursive Approach)
void postorderTraversal(struct BinaryTreeNode *root) {
    if(root == NULL) {
        return;
    }
    else {
        postorderTraversal(root->left);
        postorderTraversal(root->right);
        printf(" %d ", root->info);
    }
}

File: Tree/Tree Traversal/Postorder Traversal (Recursive)/sol.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution:
    def postorderTraversal(self, root: TreeNode) -> List[int]:
        # base case
        if not root:
            return []
        else:
            return self.postorderTraversal(root.left)+self.postorderTraversal(root.right)+[root.val]
